site-src/v1alpha2/guides/getting-started.md

```diff
@@ -21,33 +21,43 @@ Gateway API. By installing a Gateway controller in your Kubernetes cluster,
 you can try out the guides above. This will demonstrate that the desired routing
 configuration is actually being implemented by your Gateway resources (and the
 network infrastructure that your Gateway resources represent). Note that many
-of the Gateway controller setups will install and remove the Gateway API CRDs
+of the Gateway controller setups will install and remove the Gateway API bundle
 for you.
 
-## Installing Gateway API CRDs manually
+## Installing a Gateway API Bundle
 
-The following command will install the Gateway API CRDs. This includes the
+A Gateway API bundle represents the set of CRDs and validating webhook
+associated with a version of Gateway API.
+
+### Install the CRDs
+
+The following command will install the Gateway API CRDs. This includes
 GatewayClass, Gateway, HTTPRoute, TCPRoute, and more. Note that a running
 Gateway controller in your Kubernetes cluster is required to actually act on
 these resources. Installing the CRDs will just allow you to see and apply the
-resources, though they won't do anything.
+resources, but they won't do anything without a controller implementing them.
 
 ```
-kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.0" \
-| kubectl apply -f -
+kubectl apply -k "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.2"
 ```
 
-After you're done, you can clean up after yourself by uninstalling the
-Gateway API CRDs. The following command will remove all GatewayClass, Gateway,
-and associated resources in your cluster. If these resources are in-use or
-if they were installed by a Gateway controller, then do not uninstall them.
-This will uninstall the Gateway API CRDs for the entire cluster. Do not do
-this if they might be in-use by someone else as this will break anything using
-these resources.
+### Install the Webhook
 
+The validating webhook included with Gateway API is still in active development
```

Comment:
Shouldn't this be more for release notes? It's going to be hard to find this paragraph and remove it later.

Reply:
We already have an issue to track improving the installation process (https://github.com/kubernetes-sigs/gateway-api/issues/1017) that @youngnick is working on. I'll update that issue to ensure it includes updating this text. I think our next release will include those fixes and can also be used to update this text. The latest released version of the webhook (v0.4.2) does have noticeable bugs and omissions, so I think it's important to leave this in until we've resolved them as part of the v1b1 milestone.